test/ossl_shim/packeted_bio.h

```diff
@@ -10,18 +10,10 @@
 #ifndef HEADER_PACKETED_BIO
 #define HEADER_PACKETED_BIO
 
+#include "e_os.h"
```

Comment:
This bit seems to cause one of the Travis builds to be sad, e.g., https://travis-ci.org/openssl/openssl/jobs/270826141 .
Maybe C++ has different search rules for double-quote includes than C does, so that test/ossl_shim/build.info should INCLUDE ../.. as well?

Reply:
@kaduk, +1 if you do a PR to do that.  Let's fix the builds asap :)

Reply:
#4316 existed before you commented :)

Reply:
(upon further reflection the C++ bit is probably a red herring)